terraform/eks/daemon/main.tf

```diff
@@ -423,3 +424,14 @@ resource "null_resource" "validator" {
     EOT
   }
 }
+
+resource "null_resource" "modify_hop_limit" {
+  depends_on = [
+    aws_eks_node_group.this,
+  ]
+  provisioner "local-exec" {
+    command = <<-EOT
+      go run ./modify_hop_limit ${aws_eks_cluster.this.name} ${var.hop_limit}
```

Comment:
I was having issues creating a launch template so I created a function that will change the hop limit after nodes are already created